src/game.cpp

```diff
@@ -7333,24 +7328,16 @@ look_around_result game::look_around(
             creature_tracker &creatures = get_creature_tracker();
             monster *const mon = creatures.creature_at<monster>( lp, true );
             if( mon && u.sees( *mon ) ) {
-                std::string mon_name_text = string_format( _( "%s - %s" ),
-                                            ctxt.get_desc( "CHANGE_MONSTER_NAME" ),
-                                            ctxt.get_action_name( "CHANGE_MONSTER_NAME" ) );
-                mvwprintz( w_info, point( 1, getmaxy( w_info ) - 2 ), c_red, mon_name_text );
+                print_colored_text( w_info, point( 1, getmaxy( w_info ) - 2 ),
+                                    ctxt.get_hint( "CHANGE_MONSTER_NAME" ) );
             }
 
-            std::string fast_scroll_text = string_format( _( "%s - %s" ),
-                                           ctxt.get_desc( "TOGGLE_FAST_SCROLL" ),
-                                           ctxt.get_action_name( "TOGGLE_FAST_SCROLL" ) );
-            mvwprintz( w_info, point( 1, getmaxy( w_info ) - 1 ), fast_scroll ? c_light_green : c_green,
-                       fast_scroll_text );
+            print_colored_text( w_info, point( 1, getmaxy( w_info ) - 1 ), ctxt.get_hint( "TOGGLE_FAST_SCROLL",
+                                fast_scroll ?  keybinding_hint_state::TOGGLED_ON : keybinding_hint_state::TOGGLED_OFF ) );
 
             if( !ctxt.keys_bound_to( "toggle_pixel_minimap" ).empty() ) {
-                std::string pixel_minimap_text = string_format( _( "%s - %s" ),
-                                                 ctxt.get_desc( "toggle_pixel_minimap" ),
-                                                 ctxt.get_action_name( "toggle_pixel_minimap" ) );
-                right_print( w_info, getmaxy( w_info ) - 1, 1, pixel_minimap_option ? c_light_green : c_green,
-                             pixel_minimap_text );
+                right_print( w_info, getmaxy( w_info ) - 1, 1, ctxt.get_hint( "toggle_pixel_minimap",
+                             pixel_minimap_option ? keybinding_hint_state::TOGGLED_ON : keybinding_hint_state::TOGGLED_OFF ) );
             }
```

Comment:
I added an ifdef above to solve this problem
```
#if defined(TILES)
    ctxt.register_action( "toggle_pixel_minimap" );
#endif
```

when playing on curses, the keybind doesn't show up in the keybinds menu or this menu.

NB: This feature is also broken https://github.com/CleverRaven/Cataclysm-DDA/issues/58776 so that's something we'll tackle in another PR.